nntrainer/tensor/bcq_tensor.cpp

```diff
@@ -292,6 +292,22 @@ std::vector<unsigned int> BCQTensor::argmax() const {
   return result;
 }
 
+std::vector<unsigned int> BCQTensor::argmin() const {
+  std::vector<unsigned int> result;
+  const uint32_t *data = (uint32_t *)getData();
+  size_t batch_size = batch();
+  size_t feature_len = dim.getFeatureLen();
+
+  result.resize(batch_size);
+
+  for (unsigned int b = 0; b < batch_size; b++) {
```

Comment:
Since the type of batch_size is size_t, I think it's better for b to also be of type size_t.

```suggestion
  for (size_t b = 0; b < batch_size; b++) {
```

Reply:
I updated it. PTAL